pynecone/.templates/web/utils/state.js

```diff
@@ -123,8 +123,16 @@ export const updateState = async (state, result, setResult, router, socket) => {
  * @param setResult The function to set the result.
  * @param endpoint The endpoint to connect to.
  */
-export const connect = async (socket, state, setResult, endpoint) => {
+export const connect = async (socket, state, result, setResult, router, endpoint) => {
+  // Create the socket.
   socket.current = new WebSocket(endpoint);
```

Comment:
Will we eventually change this so its on socket per app

Reply:
I'm not sure tbh, we will need to investigate this.